wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/GetFeatureConfigsActionHandler.swift

```diff
@@ -159,14 +159,6 @@ final class GetFeatureConfigsActionHandler: ActionHandler<GetFeatureConfigsActio
             )
         }
 
-        if let mlsMigration = payload.mlsMigration {
-            featureRepository.storeMLSMigration(
-                Feature.MLSMigration(
-                    status: mlsMigration.status,
-                    config: mlsMigration.config
-                )
-            )
-        }
```

Comment:
Why is the MLS migration feature configuration removed in this PR?

Reply:
It was duplicated, likely due to some merge conflicts